src/round.js

```diff
@@ -0,0 +1,40 @@
+class Round {
+  constructor(players, data, puzzleBank) {
+    this.players = players;
+    this.wheelData = [...data.wheel];
+    this.puzzleBank = puzzleBank;
+    this.currentPlayer = 0;
+    this.currentPuzzle = {};
+  }
+
+  getCurrentPlayer() {
+    return this.players[this.currentPlayer];
+  }
+
+  randomizeWheel() {
+    let temporaryValue;
+    let randomIndex;
+    let currentIndex = this.wheelData.length;
+    // While there remain elements to shuffle...
+    while (0 !== currentIndex) {
```

Comment:
This condition is written a little backwards. Normally you'll see conditionals written like `currentIndex !== 0` rather than the other way around. You could likely also just check `while (currentIndex)` as 0 is a falsey value in JavaScript. And as long as there is a currentIndex value that's not 0, we can assume it's true.